dockerfiles/sv-base/Dockerfile

```diff
@@ -7,6 +7,33 @@ FROM ${MINIBASE_IMAGE}
 
 ENV HOME=/root
 
+####################################################################################################
+# GATK needed for PrintSVEvidence
```

Comment:
It might be helpful to make a ticket to update to a gatk release rather than this commit once PrintSVEvidence is merged into GATK master.

Reply:
Created #81